### PR TITLE
DLP: Add sample for update trigger and inspect data to hybrid job tri…

### PR DESCRIPTION
**Pull request by dinesh-crest**
Added unit test cases for same

## Description

References:- https://cloud.google.com/dlp/docs/creating-job-triggers#update_an_existing_job_trigger
https://cloud.google.com/dlp/docs/how-to-hybrid-jobs#formatting

Note: Before submitting a pull request, please open an issue for discussion if you are not associated with Google.

## Checklist
- [x] I have followed guidelines from [CONTRIBUTING.MD](https://github.com/GoogleCloudPlatform/nodejs-docs-samples/blob/main/CONTRIBUTING.md) and [Samples Style Guide](https://googlecloudplatform.github.io/samples-style-guide/)
- [x] **Tests** pass:   `npm test` (see [Testing](https://github.com/GoogleCloudPlatform/nodejs-docs-samples/blob/main/CONTRIBUTING.md#run-the-tests-for-a-single-sample))
- [x] **Lint** pass:   `npm run lint` (see [Style](https://github.com/GoogleCloudPlatform/nodejs-docs-samples/blob/main/CONTRIBUTING.md#style))
- [ ] These samples need a new **API enabled** in testing projects to pass (let us know which ones)
- [ ] These samples need a new/updated **env vars** in testing projects set to pass (let us know which ones)
- [ ] This sample adds a new sample directory, and I updated the [CODEOWNERS file](https://github.com/GoogleCloudPlatform/nodejs-docs-samples/blob/main/CODEOWNERS) with the codeowners for this sample
- [ ] This sample adds a new sample directory, and I created [GitHub Actions workflow](https://github.com/GoogleCloudPlatform/nodejs-docs-samples/blob/main/CONTRIBUTING.md#adding-new-samples) for this sample
- [ ] This sample adds a new **Product API**, and I updated the [Blunderbuss issue/PR auto-assigner](https://github.com/GoogleCloudPlatform/nodejs-docs-samples/blob/main/.github/blunderbuss.yml) with the codeowners for this sample
- [x] Please **merge** this PR for me once it is approved

### CODE REVIEW

**Comment by pattishin** [comment]
Hi @dinesh-crest, due to the way we have these setup - the dlp test will require that you create your branches directly against the `nodejs-docs-samples` repo and not from a forked repo. This is one of the reasons why the `dlp/test` check is failing early.

File: dlp/patchTrigger.js [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+  // Instantiates a client
+  const dlpClient = new DLP.DlpServiceClient();
+
+  // The project ID to run the API call under
+  // const projectId = 'my-project';
+
+  // The job trigger ID to run the API call under
+  // const jobTriggerName = 'your-job-trigger-name';
+
+  async function patchTrigger() {
```
Comment (jakubrauch) [inline dlp/patchTrigger.js:35]: Can we call this update to stay consistent with the rest of the example?
Same question regarding filename

File: dlp/system-test/metadata.test.js [hunk 2]
```diff
@@ -19,13 +19,15 @@ const {describe, it, before} = require('mocha');
...
+const {Storage} = require('@google-cloud/storage');
 
 const dataProject = 'bigquery-public-data';
 const dataSetId = 'samples';
 const tableId = 'github_nested';
 const fieldId = 'url';
 
-const bucketName = process.env.BUCKET_NAME;
+const storage = new Storage();
+const bucketName = `test-bucket-${uuid.v4()}`;
```
Comment (jakubrauch) [inline dlp/system-test/metadata.test.js:30]: Can we include the name of this test in the name to make it easier to track down the culprit test if these buckets stop being automatically cleaned up?

Comment (dinesh-crest) [reply dlp/system-test/metadata.test.js]: @jakubrauch Resolved the comments, thanks for pointing out. Re-requested for review

**Comment by pattishin** [comment]
> Hi @dinesh-crest, due to the way we have these setup - the dlp test will require that you create your branches directly against the `nodejs-docs-samples` repo and not from a forked repo. This is one of the reasons why the `dlp/test` check is failing early.

Hi @dinesh-crest, 
When you are able, please close and create a new branch directly off of this repo with your work.

**Pull request closed without merging**